tools/toit.cmake

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2019 Toitware ApS.
+#
+# This library is free software; you can redistribute it and/or
```

Comment:
I think we normally use a more liberal license for tools.

Reply:
For simplicity I will just go with MIT here. No need to add any other license file then.
done.